doc/source/io.rst

```diff
@@ -4018,6 +4019,43 @@ easy conversion to and from pandas.
 
 .. _xray: http://xray.readthedocs.org/
 
+.. _io.sas:
+
+SAS Format
+----------
+
+.. versionadded:: 0.17.0
+
+The top-level function ``read_sas`` currently can read (but not write)
```

Comment:
can you me the read_sas into ``` :func:`read_sas` ``` ? (then it is automatically a link to the docstring page